search/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Ah I see why you had the top level `build.gradle` now. Do we need it separated like this now that a repo is essentially one service?